pkg/util/globalerror/errors.go

```diff
@@ -46,7 +46,7 @@ const (
 
 	MetricMetadataMissingMetricName ID = "metadata-missing-metric-name"
 	MetricMetadataMetricNameTooLong ID = "metric-name-too-long"
-	MetricMetadataHelpTooLong       ID = "help-too-long"
+	MetricMetadataHelpTooLong       ID = "help-too-long" // unused, left here to prevent reuse for different purpose
```

Comment:
Please remove the related runbook "err-mimir-help-too-long" from `docs/sources/operators-guide/mimir-runbooks/_index.md`.

Reply:
done